Makd.mak

```diff
@@ -640,19 +640,19 @@ fasttest: $(fasttest)
 
 
 .PHONY: d2conv
-d2conv: $O/d2conv.stamp \
-	$(foreach s,$(SUBMODULES),$O/d2conv/$s.stamp)
+d2conv: $O/d2conv.stamp
 
-$O/d2conv.stamp: $C
-	$(call exec,git ls-files | grep '\.d$$' | \
-		xargs --no-run-if-empty d1to2fix,.,d1to2fix)
-
-$O/d2conv/%.stamp: $C/%
-	$(call exec,cd $<; git ls-files | grep '\.d$$' | \
-		xargs --no-run-if-empty d1to2fix,$<,d1to2fix)
-	$Vmkdir -p $(dir $@)
+$O/d2conv.stamp: $O/d2conv.list
+	$(call exec, d1to2fix --fatal --input=$^, $^)
```

Comment:
You should use `$<` instead, you only want the first pre-requisite (just in case you add more in the future)